### PR TITLE
Fix Countdown Edge App Screen Compatibility 

### PR DESCRIPTION
**Pull request by salmanfarisvp**
- [X] Added support of multiple screen size and orientation

### CODE REVIEW

**Comment by vpetersson** [comment]
Please add screenshots for easier review.

**Comment by salmanfarisvp** [comment]
- [X] Header to the centre 
image

- [X] Added Media query to support portrait orientation and different sizes. 

image

image

image

image

cc @vpetersson

**Comment by vpetersson** [comment]
Please make the header bigger

**Comment by salmanfarisvp** [comment]
Sure. Working on it.

**Comment by salmanfarisvp** [comment]
@vpetersson Is this size fine? 

image

**Comment by vpetersson** [comment]
LGTM, but should probably be dynamically sized. Also, the AM/PM vs 24h clock should be fixed, but that can wait.

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 
+Copyright (c) 2024
```
Comment (vpetersson) [inline LICENSE:3]: This should be reverted. It's not our copyright.

**Review by vpetersson — APPROVED** [review]

**Comment by salmanfarisvp** [comment]
> LGTM, but should probably be dynamically sized. Also, the AM/PM vs 24h clock should be fixed, but that can wait.

it's based on the Locale setting and user can override via settings.

File: LICENSE (hunk 1 above)
Comment (salmanfarisvp) [reply LICENSE]: I see. I fixed it now. https://github.com/Screenly/Playground/pull/162/commits/f456ba38d5ae82290e3182acf378b6b949ca3595

**Comment by vpetersson** [comment]
> > LGTM, but should probably be dynamically sized. Also, the AM/PM vs 24h clock should be fixed, but that can wait.
> 
> it's based on the Locale setting and user can override via settings.

Something is broken then because mine is showing up as am/pm, but we can fix that later.

**Pull request merged**